compiler/test/dotc/run-test-pickling.blacklist

```diff
@@ -6,3 +6,7 @@ t7374
 tuples1.scala
 tuples1a.scala
 typeclass-derivation2.scala
+typeclass-derivation2a.scala
+typeclass-derivation3.scala
+derive-generic.scala
```

Comment:
So far we have upgraded Tasty printing (I believe this is the culprit for the failing tests here) in a separate commit. It would be good to change that scheme, but I would really like to get more commonality between Tasty and normal printing before we do so. I find it wasteful to have two printers with completely different architectures.